spec/appliance_console/cli_spec.rb

```diff
@@ -227,6 +227,48 @@
     end
   end
 
+  context "#set_server_state" do
+    let(:evm_service) { double("evmserved") }
+    before do
+      allow(LinuxAdmin::Service).to receive(:new).with("evmserverd").and_return(evm_service)
+    end
+    it "state is status" do
+      expect(subject).to receive(:say)
+      expect(evm_service).to receive(:running?).and_return(true)
+      subject.parse(%w(--server status)).run
+    end
+
+    it "state is start" do
+      expect(subject).to receive(:say)
+      expect(evm_service).to receive(:running?).and_return(true)
+      expect(evm_service).to receive(:start)
+      evm_service.start
```

Comment:
Why are you calling methods on a double in all of these tests?

Don't do that.